lighthouse-core/gather/driver.js

```diff
@@ -428,18 +380,20 @@ class Driver {
    * @private
    * @template {keyof LH.CrdpCommands} C
    * @param {C} method
+   * @param {string=} sessionId
    * @param {LH.CrdpCommands[C]['paramsType']} params
    * @return {Promise<LH.CrdpCommands[C]['returnType']>}
    */
-  _innerSendCommand(method, ...params) {
+  _innerSendCommand(method, sessionId, ...params) {
```

Comment:
I feel like it's time to rename this to `_sendCommandWithoutTimeout`

Reply:
I like this, let's do it in a follow up though.